### DISCUSSION TITLE
FFmpeg not installing

### DISCUSSION THREAD

**Discussion by Skelly-debug** [Androz2091/discord-player#782] [Q&A]
When I start the bot it says FFmpeg is not installed even tho I typed npm install... please help.

 - Node Version: 16.9.0
 - Discord Player Version: idk
 - Discord.js Version: 13

**Comment by pateckii** [comment]
Did you 'cd' to the folder with bot files before 'npm install ffmpeg-static'?

**Reply by Skelly-debug** [reply]
yeah and still doesnt work

**Comment by AustinFRG** [comment]
I'm having the same issue, still haven't gotten it resolved.
https://github.com/Androz2091/discord-player/discussions/631

**Reply by Skelly-debug** [reply]
I got no help and i've asked several people

**Reply by AustinFRG** [reply]
Yeah, I still haven't gotten it fixed either.

**Comment by H4RDY-DEV** [comment]
I have the same issue.

**Comment by Majoramari** [comment]
**Installation:**
Try install it with [chocolatey](https://chocolatey.org/): `choco install ffmpeg`

Now type `ffmpeg` in your terminal if you saw output telling you the version and some other staff you should be ready to go.

**Comment by AustinFRG** [comment]
It already does that when I type ffmpeg.


On Sunday, November 14, 2021, 6:11 PM, MajorAmari ***@***.***> wrote:




Installation:
Try install it with chocolatey: choco install ffmpeg

Now type ffmpeg in your terminal if you saw output telling you the version and some other staff you should be ready to go.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by Majoramari** [reply]
Are you using WSL?

**Comment by AustinFRG** [comment]
I think so, yeah.


On Sunday, November 14, 2021, 11:23 PM, MajorAmari ***@***.***> wrote:




Are you using WSL?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
